apps/web/src/services/dashboards/widgets/_configs/widget-options-schema.ts

```diff
@@ -127,5 +127,36 @@ export const ASSET_WIDGET_OPTIONS_SCHEMA_PROPERTIES: Record<string, WidgetOption
             { type: 'MANAGED', key: MANAGED_VARIABLE_MODEL_CONFIGS.asset_account.key },
         ],
     },
+    asset_default_field: {
```

Comment:
question: Is there no `MANAGED_VARIABLE_MODEL_CONFIGS.asset_default_field.key`?
like `MANAGED_VARIABLE_MODEL_CONFIGS.cost_default_field.key`